mtp_noms_ops/apps/security/views/check.py

```diff
@@ -113,6 +116,37 @@ def get_form_kwargs(self):
         )
         return form_kwargs
 
+    @staticmethod
+    def get_latest_auto_accept_state(auto_accept_rule):
+        return sorted(
+            auto_accept_rule['states'], key=lambda x: x['created']
+        )[-1]
+
+    def get_unbound_active_auto_accept_state(
+        self, api_session, debit_card_sender_details_id: int, prisoner_profile_id: int
+    ) -> Optional[dict]:
+        query_existing_auto_accept_rule = api_session.get(
+            '/security/checks/auto-accept?{}'.format(
+                urlencode((
+                    ('prisoner_profile_id', prisoner_profile_id),
+                    ('debit_card_sender_details_id', debit_card_sender_details_id)
+                ))
+            )
+        )
+
+        payload = query_existing_auto_accept_rule.json()
+        #  unfortunately django-filters OR's filters rather than ANDing them
```

Comment:
(i though i left this comment somewhere, so maybe this is a duplicate or the other got lost 🤷)
i think we'll need to add this OR-ing filter to the api view. it'll be useful for the auto-accept history view too

Reply:
You mean AND-ing? (Django Filters is already set up on the API view and it's default behavior is to OR). Seems reasonable, I can't remember how to create a django-filter that filters more than one field off the top of my head, but I'm sure it's possible.

Reply:
yes you're right, sorry i meant AND-ing. i'm imagining a new `django_filters.FilterSet` subclass or something. could even be a custom GET param that the view responds to by pre-filtering the queryset